site-scrapers/PriceChopper.js

```diff
@@ -0,0 +1,50 @@
+const sites = require("../data/sites.json");
+const https = require("https");
+
+module.exports = async function GetAvailableAppointments(browser) {
+    console.log("PriceChopper starting.");
+    const webData = await ScrapeWebsiteData(browser);
+    console.log("PriceChopper done.");
+    return sites.PriceChopper.locations.map((loc) => {
+        const newLoc = { ...loc };
+        const response = webData[loc.id];
+        let hasAvailability = response[0].visibleTimeSlots.length ? 1 : 0;
```

Comment:
2 comments:
1. As I mentioned before, we can't assume that we just want to look at the first entry in the response.
2. We can fill out the specific availabilities per data given what we see in the response (take visibleTimeSlots and sum the available slots for each day). The object we want to save will look like this:
```
newLoc.availability = {
    "02/13/2021": {
         hasAvailability: true,
         numberAvailableAppointments: 10
    },
    "02/14/2021": {...},
    ...
};
```

Reply:
The response sent back is an array with one key. response[0] was the only way to reference the data. I'll try to get the actual number of appts.

Reply:
https://scrcxp.pdhi.com/ScreeningEvent/e047c75c-a431-41a8-8383-81613f39dd55/GetLocations/01610?state=MA has two stores and therefore returns an array of 2 objects.

Reply:
Looping over that array now.